tests/terraform/graph/graph_builder/test_graph_builder.py

```diff
@@ -316,9 +316,9 @@ def test_nested_modules_address_attribute(self):
         graph_manager = TerraformGraphManager(NetworkxConnector())
         local_graph, _ = graph_manager.build_graph_from_source_directory(resources_dir, render_variables=True)
         module_1 = self.get_vertex_by_name_and_type(local_graph, BlockType.MODULE, 'inner_s3_module')
-        assert module_1.attributes.get(CustomAttributes.TF_RESOURCE_ADDRESS) == 'module.s3_module.inner_s3_module'
+        assert module_1.attributes.get(CustomAttributes.TF_RESOURCE_ADDRESS) == 'module.s3_module.module.inner_s3_module'
```

Comment:
looks like a scary change I have to say - are we sure that we handle this correctly now that we have multiple `module`?

Reply:
I assume so as can be seen in the `expected_local_graph.json` file I adjusted. multi module looks good and also resource inside that module didn't change.